### PR TITLE
Fix missing nvidia-ml-py

### PR DESCRIPTION
**Pull request by leszko**

### CODE REVIEW

File: runner/docker/Dockerfile.live-base-comfyui [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN cd /comfyui/custom_nodes && \
 RUN pip uninstall -y tensorrt && \
     pip install tensorrt==10.6.0
 
-RUN pip install torch==2.5.1 torchvision torchaudio tqdm
+RUN pip install torch==2.5.1 torchvision torchaudio tqdm nvidia-ml-py==12.560.30 \
```
Comment (pwilczynskiclearcode) [inline runner/docker/Dockerfile.live-base-comfyui:28]: few lines below we install `-r requirements.txt`.
Why do we install manually and using `-r` at the same time?

**Review by pwilczynskiclearcode — APPROVED** [review]

**Review by mjh1 — APPROVED** [review]

File: runner/docker/Dockerfile.live-base-comfyui (hunk 1 above)
Comment (eliteprox) [reply runner/docker/Dockerfile.live-base-comfyui]: This was already solved in #305 using requirements files. Both options are valid, but we should go with one other the other. https://github.com/livepeer/ai-worker/commit/dbefe4201a7153983479a6d15bdfaf46687c8330

Comment (leszko) [reply runner/docker/Dockerfile.live-base-comfyui]: I believe it didn't work with my fix. You can check, but the images were failing for us in staging.

Comment (rickstaa) [reply runner/docker/Dockerfile.live-base-comfyui]: @leszko could you check if https://github.com/livepeer/ai-worker/pull/319/files works? If not then there is a dependency chain preventing the right version to be installed and we shuold only specify it in the docker container.

**Pull request merged**